NetKAN/ClassicStockResources.netkan

```diff
@@ -1,23 +1,19 @@
-{
-    "spec_version": "v1.4",
-    "identifier":   "ClassicStockResources",
-    "author":       "Angel-125",
-    "$kref":        "#/ckan/github/Angel-125/ClassicStockResources",
-    "$vref":        "#/ckan/ksp-avc",
-    "license":      "GPL-3.0",
-    "resources": {
-        "homepage": "https://github.com/Angel-125/ClassicStockResources"
-    },
-    "tags": [
-        "library",
-        "resources"
-    ],
-    "depends": [
-        { "name": "ModuleManager"                  },
-        { "name": "WildBlue-PlayMode-ClassicStock" }
-    ],
-    "install": [ {
-        "find":       "ClassicStockResources",
-        "install_to": "GameData/WildBlueIndustries"
-    } ]
-}
+spec_version: v1.4
+identifier: ClassicStockResources
+author: Angel-125
+$kref: '#/ckan/github/Angel-125/ClassicStockResources'
+$vref: '#/ckan/ksp-avc'
+license: GPL-3.0
+resources:
+  homepage: https://github.com/Angel-125/ClassicStockResources
+tags:
+  - library
+  - resources
+conflicts:
+  - name: CommunityResourcePack
+depends:
+  - name: ModuleManager
```

Comment:
man it's very close to not even needing MM, but there are a couple `:NEEDS[!CommunityResourcePack]` in there